python/cudf_polars/cudf_polars/typing/__init__.py

```diff
@@ -13,9 +13,7 @@
 from polars.polars import _expr_nodes as pl_expr, _ir_nodes as pl_ir
 
 if TYPE_CHECKING:
-    from typing import Callable
-
-    from typing_extensions import TypeAlias
+    from typing import Callable, TypeAlias
```

Comment:
`TypeAlias` moved to `typing` in Python 3.10: https://docs.python.org/3/library/typing.html#typing.TypeAlias

There are still other imports from `typing_extensions` in `cudf-polars` though, so we can't drop the dependency on it.